stdlib_test.go

```diff
@@ -122,140 +122,21 @@ func TestStdLibAdapterPrefixedExtractionWithJoinToMessage(t *testing.T) {
 	}
 }
 
-func TestStdlibAdapterSubexps(t *testing.T) {
```

Comment:
Was this removed by accident?

Reply:
I was not aware of how to test private methods